### DISCUSSION TITLE
Inline element style with css and tw prop

### DISCUSSION THREAD

**Discussion by Limunce** [ben-rogerson/twin.macro#330] [Q&A] [answered]
Hello guys, I just run into a strange issue. I'm not able to style div element with css prop.

` <div
      css={css`
        ${tw`grid gap-x-20 gap-y-23 my-20 w-full`};
        grid-template-columns: repeat(auto-fill, minmax(355px, 1fr));
      `}
    >
      Some text
    </div>`

In the inspector  I see this : 
![Screenshot 2021-02-10 at 16 06 40](https://user-images.githubusercontent.com/34006580/107528499-33ad0c00-6bba-11eb-99cd-1454687c5a9c.png)

And the strange thing is that this happens only in some components 😕 

The same code works as it should in another component.

I'm using twin for a long time, and this is the first time I run on an issue like this.

Does anyone know what is the problem here?

Thank you in advance :)

EDIT
Hmm, I found what is the problem.

I cannot use the css prop like this

```
<div css={css`
${tw`bg-red`};
`}
>
Tesst
</div>

```

because I'm not using tw="..." prop anywhere in that component...
I think this was not the case with older versions. I'm now on the latest one.

@ben-rogerson Am I missing something or this is normal behavior? 😕

**Comment by ben-rogerson** [comment, marked as answer]
The semicolon after the interpolated tw call is causing issues with the output (in styled-components).

This works:

```js
<div
  css={css`
    ${tw`block`}
    margin-top: 5px;
  `}
>
  Tesst
</div>;
```

I made some ordering changes a few versions ago so perhaps that's caused this.
Perhaps I can sniff for a semicolon after the block and remove it with twin.

**Reply by Limunce** [reply]
I just tried the way you said and got the same result :S

Idk, maybe it is my project that causes problems... I'll try with the fresh CRA project. 

Anyway, thank you so much for your work! ❤️

**Reply by ben-rogerson** [reply]
If you don't have any luck then perhaps you could cruise down a few versions in twin/styled-components to see when the change was made?

**Reply by Limunce** [reply]
I tried with the fresh CRA and the latest version of twin. Everything is working fine, so obviously, my project causing this problem.
Idk why but I'll try to find out. :)

Thank you for your time :)
